RULES.md

```diff
@@ -0,0 +1,94 @@
+
+### Products
+
+- Keeping track of the decomposition: Metadata in product.yml files, CODEOWNERS file
+- Area of effect: All PSPs that don’t have specific stakeholders, associated content and test scenarios.
+- Guidelines: Care needs to be taken when content requires a slightly different approach for different products - don’t copy paste, and also don’t change existing behavior for everybody (link to coding guidelines needed).
+
+
+### Product Specific Profiles (PSPs)
+
+- Keeping track of the decomposition: Metadata in profile yml files, CODEOWNERS file
+- Area of effect: Respective profile files
+- Guidelines: N/A
+
+
+### Product Independent Profiles (PIPs)
+
+- Keeping track of the decomposition: Metadata in control files. Any profile definitions that live as separate profile files are simply a product’s content.
+- Area of effect: Just control files
+- Guidelines: Let’s try and see whether the control file format helps us with effort sharing.
+
+
+### Build system
+
+- Keeping track of the decomposition: CODEOWNERS file
+- Area of effect: Everything that is directly or indirectly used during the content build process
+- Guidelines: We need strong protection against technical debt, and a framework to deal with deprecations or changes of behavior.
+
+
+### Shared Resources
+
+- Keeping track of the decomposition: CODEOWNERS file
+- Area of effect: Content that impacts more products.
+- Guidelines: The same as for the build system.
+
+
+### Tests
+
+- Keeping track of the decomposition: CODEOWNERS file
+- Area of effect: Test scenarios, test suite features, unit tests, gating tests
+- Guidelines: We need guidelines to preserve test robustness and the rate of tests usefulness.
+
+
+### Organizational administration
+
+- Keeping track of the decomposition: Maintainer file, readme?
+- Area of effect: Github org and project settings
+- Guidelines: Enter and exit conditions
+
+
+## Rights assignment
+
+### Individual PR approve and merge rights
+
+TLDR: 6 non-trivial PRs within 6 months => merge rights for 1 year since the last activity.
+
+Contributors can ask for merge rights on the mailing list or on Gitter.
+Although one can technically merge code, the code contribution guidelines still have to be obeyed.
+
+Recommended way to get merge rights: Anybody who submits six non-trivial PRs that get merged in a period no longer than six months are encouraged to ask for rights.
+A person can ask to be stripped of merge rights.
+If a person doesn’t actively contribute to the project for a period of 12 months, their merge rights are stripped. Active contribution consists of either non-trivial contributions of code, non-trivial reviews, or analyses of problems in issues.  Rights can be also revoked if coding style or conduct are consistently broken.
+The project maintainers decide about granting or strip of rights and about exceptions to the procedure e.g. when an applicant has deep prior experience with the project.
+
+
+### Organizations
```

Comment:
I'm not sure I really see how the below paragraph meshes with the above. Will e.g., SUSE have only a single merge-bits maintainer (+ 1 backup)? Or if they had multiple SUSE-employed contributors above the threshold, they'll get all of them under the above 6 month term limit plus one (+ 1 backup) "long term" 12-month merge bits people?

Reply:
The idea is that an organization will get one more person with merge access "for free". This and the project administration are actually the only places where the "organization" concept is effective.